web/src/lib/components/Button.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+	export let onClick: () => void = () => {};
+	export let type: 'button' | 'reset' | 'submit' = 'button';
+	export let isDisabled: boolean = false;
+	let className: string = "";
+	export { className as class };
```

Comment:
ah no it gets used as `class` when other components reference the component

Reply:
We can't `let class: string = "";` because `class` is a keyword. The `export { className as class };` is a little trick to allow us to make it look like a regular attribute on components.